db/seeds.rb

```diff
@@ -5,3 +5,7 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+['registered', 'admin'].each do |role|
```

Comment:
Like @fiestacasey said in a different comment, if you make a constant in role.rb for admin, you should add some for registered too.